### PR TITLE
Correct description

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/8037

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Example 3 assigns the per-user online voice routing policy RedmondOnlineVoiceRou
 Grant-CsOnlineVoiceRoutingPolicy -PolicyName "RedmondOnlineVoiceRoutingPolicy" -Global
 ```
 
-Example 4 assigns the per-user online voice routing policy RedmondOnlineVoiceRoutingPolicy to all the users in the tenant, except any that have an explicit policy assignment.
+Example 4 assigns the per-user online voice routing policy RedmondOnlineVoiceRoutingPolicy as the Global Online Voice Routing Policy, this affects all the users in the tenant, except any that have an explicit policy assignment.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:68]:
```suggestion
Example 4 assigns the per-user online voice routing policy RedmondOnlineVoiceRoutingPolicy as the global online voice routing policy. This affects all the users in the tenant, except any that have an explicit policy assignment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**